### PR TITLE
chore(setup-material-UI):Initialize a material UI theme to use for the app

### PR DESCRIPTION
**Pull request by Francoise02**
#### What does this PR do?
Initialize a material UI theme to use for the app.
It should have fonts defined and other necessary general stylings.

#### How should this be manually tested?

1. Clone this repository
2. Switch to this branch  ch-setup-materialUI-#182923295
3. Install dependencies by running `yarn` command
4. Start the server; `yarn start`
5. You should be able to see the menu now and when you click to `themes` you can see more details about the UI theme, font, and other stylings.

#### Any background context you want to provide?

The color code used was provided on the UI/UX Figma design.

#### What are the relevant pivotal tracker stories?
[#182923295](https://www.pivotaltracker.com/story/show/182923295)

### CODE REVIEW

**Review by kalisaNkevin — APPROVED** [review]
You can remove the commented lines.

File: src/theme/theme.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    // type: 'dark',
+    primary: {
+      main: amber[500],
+    },
+    secondary: {
+      main: grey[50],
+    },
+  },
+  typography: {
+    fontFamily: 'Roboto',
```
Comment (ericus123) [inline src/theme/theme.js]: Add a second font family in case roboto takes some time to load

File: src/theme/theme.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import indigo from '@material-ui/core/colors/indigo'
+
+const theme = createMuiTheme({
+  palette: {
+    // type: 'dark',
+    primary: {
+      main: amber[500],
+    },
+    secondary: {
+      main: grey[50],
```
Comment (ericus123) [inline src/theme/theme.js]: make sure to use the colors on your mockup instead.
declare the  colors as variables and import and use them here.

**Review by ericus123 — CHANGES_REQUESTED** [review]

File: src/MenuBar/MenuBar.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+//             <MenuIcon />
+//           </IconButton>
+//           <Typography variant="h6" >
+//             Theme Example
+//           </Typography>
+//         </Toolbar>
+//       </AppBar>
+//       </div>
+//   );
+// }
```
Comment (ericus123) [inline src/MenuBar/MenuBar.js]: remove unused codes

**Review by ericus123 — CHANGES_REQUESTED** [review]

**Review by ericus123 — APPROVED** [review]

**Review by ishimweeli — APPROVED** [review]

**Pull request merged**